modules/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "modules",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "luxon": "^3.4.3"
+  }
+}
```

Comment:
- [ ] The `package.json` and `package-lock.json` files should not be in the modules folder. Also, you have two sets of these files in your codebase, please correct this.